From akpm@linux-foundation.org Mon Feb  5 16:16:37 2007
From: akpm@linux-foundation.org
Date: Mon, 05 Feb 2007 16:15:25 -0800
Subject: powerpc: make it compile for multithread change
To: greg@kroah.com
Cc: akpm@linux-foundation.org, akpm@linux-foundation.org, benh@kernel.crashing.org, paulus@samba.org
Message-ID: <200702060016.l160GSZl004048@shell0.pdx.osdl.net>


From: Andrew Morton <akpm@osdl.org>

arch/powerpc/kernel/of_platform.c:479: error: unknown field `multithread_probe' specified in initializer

Cc: Paul Mackerras <paulus@samba.org>
Cc: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 arch/powerpc/kernel/of_platform.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/arch/powerpc/kernel/of_platform.c
+++ b/arch/powerpc/kernel/of_platform.c
@@ -475,9 +475,6 @@ static struct of_platform_driver of_pci_
        .name = "of-pci",
        .match_table = of_pci_phb_ids,
        .probe = of_pci_phb_probe,
-       .driver = {
-	       .multithread_probe = 1,
-       },
 };
 
 static __init int of_pci_phb_init(void)
